src/main/java/seedu/address/model/task/DateTime.java

```diff
@@ -30,16 +35,18 @@
     public DateTime(String dateTime) {
         requireNonNull(dateTime);
         checkArgument(isValidDateTime(dateTime), MESSAGE_CONSTRAINTS);
-        value = dateTime;
+        value = LocalDateTime.parse(dateTime, FORMATTER);
+        isDefault = false;
     }
 
     /**
-     * Constructs a default {@code DateTime}.
+     * Constructs a default {@code DateTime} which is null.
```

Comment:
why do you need to specify "which is null" when the attributes are the default values?

Reply:
sry this was a mistake. I changed it. the datetime won't be null